### PR TITLE
Add breadcrumbs to article pages

### LINKED ISSUE #374: Navigate to CSS>CSS-Frameworks and the whole site layout will change
Not sure about any other case for this but encountered this problem on Linux Mint with Chrome Version 61.0.3163.100 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by Bouncey**
closes #374 

I have the component working fine, but it relies on `titleify` for formatting, which is unmaintainable. I want to find a better solution for formatting words like HTML, CSS, C++, javaScript etc... 

breadcrumbs

This PR also fixes the CSS Frameworks articles that were injecting stylesheets and javascript into the site when being viewed, and adds a CSS Frameworks index page.

### CODE REVIEW

**Comment by QuincyLarson** [comment]
@Bouncey this looks amazing. Even if there are some issues with capitalization, I still think we should merge this if you think it's otherwise functional. Then we can fix Titelify itself with a subsequent PR.

**Comment by Bouncey** [comment]
Functionally, this is fine. Once merged we can look into handling title formatting better. 👍

File: src/templateComponents/Breadcrumbs.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'gatsby-link';
+import titleify from '../../seed/utils';
```
[the remark this answers is not in the document]
Comment (Bouncey) [reply src/templateComponents/Breadcrumbs.jsx]: No Idea what happened there....

File: src/templates/Article.jsx [hunk 2]
```diff
@@ -28,9 +33,18 @@ function Article(props) {
           rel='canonical'
         />
       </Helmet>
-      <div
-        dangerouslySetInnerHTML={{ __html: html }}
-      />
+      <Row>
```
[the remark this answers is not in the document]
Comment (Bouncey) [reply src/templates/Article.jsx]: This was just a belt-and-braces way to make sure it would wrap correctly on mobile

**Comment by Bouncey** [comment]
@systimotic this is ready for another QA 👍

**Comment by QuincyLarson** [comment]
@Bouncey This looks great! Thanks for implementing this!

**Pull request merged**